sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,28 +1,18 @@
 # Release History
 
-## 1.2.0 (2023-04-26)
-Add support for computing the claims digest from Azure Confidential Ledger application claims in the `azure.confidentialledger.receipt` module.
-
-### Features Added
-- Add `compute_claims_digest` function to compute the claims digest from a list of application claims JSON objects. 
-- Add optional argument in `verify_receipt` function to accept a list of application claims and compare the claims digest in the receipt with the computed digest from the plain claims.
-- Modify sample code to get and verify a write receipt from a running Confidential Ledger instance, with the option to pass application claims in the verification function.
-- Update README with examples and documentation for application claims.
-
-### Other Changes
-- Add tests for application claims models and digest computation public method.
-
-## 1.1.0 (2022-12-21)
+## 1.1.0 (2023-05-09)
 Add `azure.confidentialledger.receipt` module for Azure Confidential Ledger write transaction receipt verification.
```

Comment:
Do you want to move this into Features Added too? (It is up to you)

Reply:
This is general summary of the new release, so it makes sense to keep it here imo. But I can move it in "Featured Added" if you think it makes more sense.

Reply:
`Features Added` section will be automatically added into release notes. That's the main difference.